spk/plexivity/src/installer.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+
+# Package
+PACKAGE="plexivity"
+DNAME="plexivity"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PYTHON_DIR="/usr/local/python"
+GIT_DIR="/usr/local/git"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${GIT_DIR}/bin:/usr/local/bin:/bin:/usr/bin:/usr/syno/bin:${PATH}"
+USER="plexivity"
+GROUP="users"
```

Comment:
nope only to it's var folder for caching images and stuff, so change the group to `nobody`?